quantum/rgblight.h

```diff
@@ -64,19 +62,10 @@
 |-----------------|-----------------------------------|
  *****/
 
-#define _RGBM_SINGLE_STATIC(sym)   RGBLIGHT_MODE_ ## sym,
-#define _RGBM_SINGLE_DYNAMIC(sym)  RGBLIGHT_MODE_ ## sym,
-#define _RGBM_MULTI_STATIC(sym)    RGBLIGHT_MODE_ ## sym,
-#define _RGBM_MULTI_DYNAMIC(sym)   RGBLIGHT_MODE_ ## sym,
-#define _RGBM_TMP_STATIC(sym)      RGBLIGHT_MODE_ ## sym,
-#define _RGBM_TMP_DYNAMIC(sym)     RGBLIGHT_MODE_ ## sym,
-enum RGBLIGHT_EFFECT_MODE {
-    RGBLIGHT_MODE_zero = 0,
-#include "rgblight.h"
-    RGBLIGHT_MODE_last
-};
 
-#ifndef RGBLIGHT_H_DUMMY_DEFINE
```

Comment:
This needs to be reverted.

This is actually required for the rgb matrix code.  If this is removed, they failed to compile.